test/extended/fixtures/test-build.json

```diff
@@ -38,6 +38,7 @@
         "strategy": {
           "type": "Source",
           "sourceStrategy": {
+            "env": [ { "name": "FOO", "value": "test" } ],
```

Comment:
note that I added this here to test that the default value "test" will be changed to "foo" when using `-e`